### ISSUE TITLE
Request Updating library (go.mod) to v 1.14

### ISSUE THREAD

**Issue by houseofcat** [streadway/amqp#458]
Please and thank you.

**Comment by bombsimon** [comment]
Changing the version in `go.mod` for this package only says that it cannot be imported in projects with an older version. If you run `v.1.14` in your project you are able to use this package without any problems.

What do you want to accomplish by bumping the version of this package's `go.mod`?

**Comment by houseofcat** [comment]
You can close this ticket out, while it is nice to keep these things up to date, the version of Golang that had the issue was in the 11.x branch and solved by the time we were in 1.12. I mistakenly thought the go directive bug was in 1.12 and wanted to ensure that I could continue moving my library, based on this repo, moving forward.

As a word of caution while it is absolutely not necessary to update the go mod directive (i.e. golang feature limitations) it doesn't really gain much by leaving it on older versions. Furthermore, updating the go mod directive and re-running unit tests (which I assume is standard practice) can give you peace of mind that the library is still in fact operating optimally on current code, allows you to use newer features, and still retaining backwards compatibility by golangs inherent design.

**Comment by bombsimon** [comment]
Yeah, you are right! Since Go only supports the latest two versions it might be a good idea to be somewhere nearer that.

I can't close any issue since I'm not an owner or member of the project. Feel free to leave it open or close it yourself, I was just looking through the issues to see if I cold help out. :)
